.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: tests
+
+on: [ pull_request ]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        python-version: [ 3.7, 3.8, 3.9 ]
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          pip install -r requirements.txt
```

Comment:
As all dependencies are within setup.py, I believe we don't need this step. I think we need something like `pip install .` instead.